### ISSUE TITLE
Updating license in metadatafile

### ISSUE THREAD

**Issue by klauswyser** [EC-Earth/ece2cmor3#759]
The [CMIP6 model data license has changed recently tp CC BY 4.0](https://pcmdi.llnl.gov/CMIP6/TermsOfUse) (removed SA). I tried updating a metadata file with a new license text:
````
CMIP6 model data produced by the EC-Earth-Consortium is licensed under a Creative Commons Attribution 4.0 International License (CC BY 4.0; https://creativecommons.org/licenses/). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing CMIP6 output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file). The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law.
````
as it is suggested by [CMIP6](https://pcmdi.llnl.gov/CMIP6/TermsOfUse). However, the CMIP6 tables don't like this and I get
````
! Error: The attribute "license" could not be validated.
[...]
! Valid values must match those found in the "license" section of your Controlled Vocab
````
I tried updating the CMIP6 tables with
````
git submodule update --init --recursive
````
but that doesn't do anything.

**Comment by treerink** [comment]
We need an update of the output control files, it is planned for Monday. At the table side we should be fine already, if everything is fine I did this update already.

**Comment by klauswyser** [comment]
It's a stupid problem. The text suggested by CMIP6 mentions the abbrevation of the license explicitely 

````...Creative Commons Attribution 4.0 International License (CC BY 4.0; https://creativecommons.org/licenses)...````

but that abbreviation isn't mentoned in CMIP6_CV.json and therefore triggers an error. Removing the abbreviation

````...Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses)...````

solves the problem yet it doesn't comply with the [example from CMIP6].(https://pcmdi.llnl.gov/CMIP6/TermsOfUse/TermsOfUse6-2.html)

**Comment by klauswyser** [comment]
And another thing: I did update my CMIP6 tables, but the ``git submodule...`` didn't help. I went to ece2cmor3/resources/tables directory and found that there was a detached HEAD. Is that normal? After switching to the master branch I could updatet he tables (fast-forward) yet it didn't help the problem with the license text.

**Comment by treerink** [comment]
At the table side we should be fine already (it took long but this was solved by last autumn):
```
                "license_info":{
                    "exceptions_contact":"@ec-earth.org <- cmip6-data",
                    "history":"2019-04-05: initially published under CC BY-SA 4.0; 2022-06-02: relaxed to CC BY 4.0",
                    "id":"CC BY 4.0",
                    "license":"Creative Commons Attribution 4.0 International (CC BY 4.0; https://creativecommons.org/licenses/by/4.0/)",
                    "source_specific_info":"",
                    "url":"https://creativecommons.org/licenses/by/4.0/"
                },
```
However, the metadata templates need an update. This will be provided in the  `r9474-output-control-files` branch. If you want to go on immediately, please update your metadata file (for cmorisation) according to the license above, that should do the trick.

**Comment by treerink** [comment]
The metadata templates are updates for the license in the `r9474-output-control-files` branch.
